Redirect install instructions to standard installfest

CheezItMan opened the pull request:
I've never liked that we had 2 installfests, and will instead have students directed to follow the textbook curriculum installfest.  

I did leave in, for now instructions for them to install chrome and that they'll need to use irb and a text editor over a repl.

kaidamasaki commented on preparing-to-code/environment-setup/README.md:
```diff
@@ -52,14 +42,6 @@ To check your version of macOS/OS X, click the Apple icon at the far left edge o
-
-We'll use the Terminal to install the _Command Line Tools_. Open Terminal and start the installation by typing `$ xcode-select --install`. Follow the instructions and wait for the installation to finish (Protip: have a snack and maybe go for a walk while it's downloading and installing).
-
-After it finishes, verify the installation by typing `xcode-select -p`. It should respond with something very similar to `/Library/Developer/CommandLineTools` (the directory where they were installed).
```
The installfest refers here for how to install the XCode Command Line Tools.

kaidamasaki commented on preparing-to-code/environment-setup/README.md:
```diff
@@ -70,52 +52,6 @@ We will use the Chrome web browser by Google in class, because it provides more
-
-Installing Homebrew involves copying and pasting `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
-
-You may need to enter your computer password to complete the installation.
```
Jumpstart is mentioned for Homebrew as well.

kaidamasaki commented on preparing-to-code/environment-setup/README.md:
```diff
@@ -70,52 +52,6 @@ We will use the Chrome web browser by Google in class, because it provides more
-* Download the zip to your Applications folder
-* Unzip the folder, by double-clicking, or right-click _Open_
-* Drag VS Code to your Applications folder and open it up
-* Press `F1` and type `Shell Command: install 'code' command in PATH`.  This will allow you to invoke VS code from the terminal. 
```
VS Code install instructions also point here.

kaidamasaki replied:
We should also probably have them use Homebrew to install VS Code.

CheezItMan commented:
@kaidamasaki Yeah I've updated the textbook installfest to match, and created a PR as well.  It also uses brew to install VS Code.  I'd thought of including instructions to use brew to install Chrome,  but they're more likely to already have it installed.

kaidamasaki commented on preparing-to-code/environment-setup/README.md:
```diff
@@ -26,21 +26,17 @@ A text editor is a computer program that allows you to create and edit files tha
+In the Ada Textbook curriculum we have an [installfest](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/installfest.md) prepared which will walk you through installing all the software required to start at Ada.
 
-#### _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
+#### _macOS Mohave_, _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
```
```suggestion
#### _macOS Mojave_, _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
```

kaidamasaki commented:
@CheezItMan Maybe add a section about Chrome that is just "if you haven't already installed Chrome run...".  (Also, for C13 we want to do Firefox but that will be a bigger change.)

CheezItMan commented:
> I didn't look at this line by line closely for grammar or clarity, but read through it and it looked good!
> 
> Also, I want to add the comment that: on my new laptop, I installed VS Code with `brew cask install visual-studio-code` and it installed it perfectly into my Applications folder.

I've added sections to install firefox and chrome via homebrew, listing Chrome as optional.  I also added a note on installing Slack via homebrew.
